bot.py

```diff
@@ -89,6 +89,7 @@ async def on_command_error(self, context: commands.Context[ShroomBot], exception
 
   async def on_message(self, message: Message):
     if message.author.bot:
+      await message.add_reaction("🍄")
```

Comment:
This will make the bot react to **every single message sent by a bot with a mushroom**, including it's own messages, so I'm probaby not gonna let this one in